kyuubi-server/src/test/scala/org/apache/kyuubi/server/api/v1/BatchesResourceSuite.scala

```diff
@@ -199,6 +203,31 @@ class BatchesResourceSuite extends KyuubiFunSuite with RestFrontendTestHelper wi
     assert(!deleteBatchResponse.readEntity(classOf[CloseBatchResponse]).isSuccess)
   }
 
+  test("open batch session with uploading resource") {
+    val requestObj = newSparkBatchRequest(Map("spark.master" -> "local"))
+    val requestStr = mapper.writeValueAsString(requestObj)
+    val exampleJarFile = {
+      val sparkProcessBuilder = new SparkProcessBuilder("kyuubi", KyuubiConf())
```

Comment:
using `sparkBatchTestResource` directly. 

See `BatchTestHelper`.

Reply:
This's a helpful hint. Addressed.